units/en/unit1/quiz.mdx

```diff
@@ -0,0 +1,168 @@
+    {
+      text: "We receive an observation when we play with chess environment",
+      explain: "We only see a part of the level close to the player.",
+      correct: true
+    }
+	]}
+/>
+
+### Q4: A task is an instance of a Reinforcement Learning problem. What are the two types of tasks?
+
+<Question
+	choices={[
+		{
+			text: "Episodic",
+			explain: "In Episodic task, we have a starting point and an ending point (a terminal state). This creates an episode: a list of States, Actions, Rewards, and new States. For instance, think about Super Mario Bros: an episode begin at the launch of a new Mario Level and ending when you’re killed or you reached the end of the level.",
+      correct: true
+		},
+    {
+			text: "Recursive",
+			explain: ""
+		},
+    {
+			text: "Adversarial",
+			explain: ""
+		},
+    {
+      text: "Continuing",
+      explain: "Continuing tasks are tasks that continue forever (no terminal state). In this case, the agent must learn how to choose the best actions and simultaneously interact with the environment.",
+      correct: true
+    }
+	]}
+/>
+
+### Q5: What is the exploration/exploitation tradeoff?
+
+<details>
+<summary>Solution</summary>
+
+In Reinforcement Learning, we need to **balance how much we explore the environment and how much we exploit what we know about the environment**.
+
+- *Exploration* is exploring the environment by **trying random actions in order to find more information about the environment**.
+
+- *Exploitation* is **exploiting known information to maximize the reward**.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/expexpltradeoff.jpg" alt="Exploration Exploitation Tradeoff" width="100%">
+
+</details>
+
+
+### Q6: What is a policy?
+
+<details>
+<summary>Solution</summary>
+
+- The Policy π **is the brain of our Agent**, it’s the function that tell us what action to take given the state we are. So it defines the agent’s behavior at a given time.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/policy_1.jpg" alt="Policy">
+
+</details>
+
+
+### Q7: What are value-based methods?
+
+<details>
+<summary>Solution</summary>
+
+- Value-based methods is one of the main approaches for solving RL problems.
+- In Value-based methods, instead of training a policy function, **we train a value function that maps a state to the expected value of being at that state**.
+
+
+
+</details>
+
+### Q8: What are policy-based methods?
+
+<details>
+<summary>Solution</summary>
+
+- In *Policy-Based Methods*, we learn a **policy function directly**.
+- This policy function will **map from each state to the best corresponding action at that state**. Or a **probability distribution over the set of possible actions at that state**.
+
+
+
+
+</details>
+
+
+Congrats on finishing this Quiz 🥳, if you missed some elements, take time to read again the chapter to reinforce (😏) your knowledge.
```

Comment:
Maybe reinforce that we'll go over these topics in depth and do practical things. Some people might be overwhelmed or scared by the number of new concepts

Reply:
:heavy_check_mark: